.buildkite/core.rayci.yml

```diff
@@ -306,11 +322,3 @@ steps:
     commands:
       - bash /c/workdir/.buildkite/windows_ci.sh test_python
     parallelism: 10
-
-  - label: ":windows: core tests"
-    tags:
-      - core_cpp
-    job_env: WINDOWS
-    instance_type: windows
-    commands:
-      - bash /c/workdir/.buildkite/windows_ci.sh test_core
```

Comment:
so can we remove the `test_core` function in `ci/ci.sh` now?

Reply:
likely, I'll take a deeper look and clean those up in another PR